docs/2023/Sunbird RC/Credential Sharing Consent and Share Credentials/updates/2023-07-21.md

```diff
@@ -5,13 +5,22 @@ author: Akshat Upadhyay
 
 ## Milestones
 
--   [ ] Give the description about Milestone 1
--   [ ] Give the description about Milestone 2
--   [ ] Give the description about Milestone 3
--   [ ] Give the description about Milestone 4
+-   [x] Made a mock portal similar to Figma Design proposed
+-   [x] Made requests through other CHAPI wallets from the form
+-   [x] Started looking into having CHAPI integrate with the credential system
+-   [x] Made documentation to be presented to others working on Avsar to work on my proposed idea
 
 ## Screenshots / Videos
 
+-   Mock Scholarship portal with Apply Using Avsar
+    ![Mock Scholarship Form](https://i.imgur.com/k3OCjjt.png)
+-   https://docs.google.com/document/d/1dcGg_KX7KUTKZDQdfdjXOU9Um1-LElhD8ZLHckCguDw/edit?usp=sharing
```

Comment:
Instead of adding the document link please add screenshots or videos

Reply:
Hey Shubhangi, I have made the new commit, kindly review. Thanks.